lib/src/tasks/local/api.dart

```diff
@@ -18,10 +18,14 @@ import 'dart:async';
 
 import 'package:dart_dev/util.dart' show TaskProcess;
 
+import 'package:dart_dev/src/tasks/config.dart';
 import 'package:dart_dev/src/tasks/task.dart';
 
 LocalTask local(String executable, Iterable<String> args) {
-  TaskProcess process = new TaskProcess(executable, args);
+  TaskProcess process = new TaskProcess(executable, args, environment: {
+    // Make a serialized version of the config available to this task.
+    'dart_dev_config': serializeConfig(),
```

Comment:
Since environment variables shouldn't hold a lot of data, I'm thinking it might be better to write the config to a file, and then make this the path to that file.

Reply:
Yeah I don't see any harm in writing it to something like `.dart_tool/dart_dev/config.json`